site/content/getting-started.md

```diff
@@ -139,7 +147,7 @@ $ curl -X PUT -i -H 'Content-Type: application/json' --data-binary '{"temp": 5}'
 or (using HTTPie):
 
 ~~~sh
-$ http PUT http://localhost:8080/registration/DEFAULT_TENANT/4711 temp:=5
+$ http PUT http://localhost:8080/telemetry/DEFAULT_TENANT/4711 temp:=5
```

Comment:
It seems the docs changes have been lost in some rebase. Do you maybe having a copy locally? I can easily write it up again